src/main/java/org/opentripplanner/index/IndexAPI.java

```diff
@@ -153,14 +156,14 @@ public Response getAgencyRoutes (@PathParam("feedId") String feedId, @PathParam(
    @Path("/stops/{stopId}")
    public Response getStop (@PathParam("stopId") String stopIdString) {
        FeedScopedId stopId = GtfsLibrary.convertIdFromString(stopIdString);
-       Stop stop = index.stopForId.get(stopId);
+       ApiStop stop = StopMapper.mapStop(index.stopForId.get(stopId));
```

Comment:
This was returning a "domain" Stop before, I cant see that the StopAdapter comes into play here. Or, did I miss something?

It seems like there is a lot of types in play for just stop on this API, but if the "domain" stop is retuned then we need to make an API type for the old `dev-1.x` Stop, too. I can discuss this @abyrd and possible the community - maybe it is better to clean it up.

Reply:
@abyrd What do you think about this. I think this is a mistake, the old `StopType` (now `ApiStop`) have the same field as the domain `Stop`, but with slightly different names. I think we should just return the `ApiStop` here, and not create another `ApiStop2` to be backward compatible.

Reply:
I apologize for this one, I managed to confuse my self - with all the different Stop objects in. use. I think what you did here is right.